lib/v2/mixcloud/index.js

```diff
@@ -0,0 +1,129 @@
sExclusive\n  isDisabledCopyright\n}\n\nfragment AudioCardProgress_cloudcast on Cloudcast {\n  id\n  proportionListened\n  audioLength\n}\n\nfragment AudioCardRepostButton_cloudcast on Cloudcast {\n  id\n  isReposted\n  isExclusive\n  isPublic\n  reposts {\n    totalCount\n  }\n  owner {\n    isViewer\n    isSubscribedTo\n    id\n  }\n}\n\nfragment AudioCardRepostButton_viewer on Viewer {\n  me {\n    id\n  }\n}\n\nfragment AudioCardShareButton_cloudcast on Cloudcast {\n  id\n  isUnlisted\n  isPublic\n  slug\n  description\n  picture {\n    urlRoot\n  }\n  owner {\n    displayName\n    isViewer\n    username\n    id\n  }\n}\n\nfragment AudioCardStats_cloudcast on Cloudcast {\n  isExclusive\n  isDraft\n  hiddenStats\n  plays\n  publishDate\n  qualityScore\n  listenerMinutes\n  owner {\n    isSubscribedTo\n    id\n  }\n  tags(country: "GLOBAL") {\n    tag {\n      name\n      slug\n      id\n    }\n  }\n  ...AudioCardTags_cloudcast\n}\n\nfragment AudioCardTags_cloudcast on Cloudcast {\n  tags(country: "GLOBAL") {\n    tag {\n      name\n      slug\n      id\n    }\n  }\n}\n\nfragment AudioCardTitle_cloudcast on Cloudcast {\n  id\n  slug\n  name\n  audioType\n  isLiveRecording\n  isExclusive\n  owner {\n    id\n    displayName\n    username\n    ...Hovercard_user\n    ...UserBadge_user\n  }\n  ...AudioCardPlayButton_cloudcast\n  ...ExclusiveCloudcastBadgeContainer_cloudcast\n  ...CloudcastHQAudio_cloudcast\n}\n\nfragment AudioCard_cloudcast on Cloudcast {\n  id\n  slug\n  name\n  audioType\n  isAwaitingAudio\n  isDraft\n  isScheduled\n  restrictedReason\n  publishDate\n  isLiveRecording\n  isDisabledCopyright\n  owner {\n    isViewer\n    username\n    id\n  }\n  picture {\n    ...UGCImage_picture\n  }\n  ...AudioCardTitle_cloudcast\n  ...AudioCardProgress_cloudcast\n  ...AudioCardActions_cloudcast\n  ...QuantcastCloudcastTracking_cloudcast\n}\n\nfragment AudioCard_viewer on Viewer {\n  ...AudioCardActions_viewer\n  me {\n    uploadLimits {\n      tracksPublishRemaining\n      showsPublishRemaining\n    }\n    id\n  }\n}\n\nfragment CloudcastHQAudio_cloudcast on Cloudcast {\n  audioQuality\n}\n\nfragment ExclusiveCloudcastBadgeContainer_cloudcast on Cloudcast {\n  isExclusive\n  isExclusivePreviewOnly\n  slug\n  id\n  owner {\n    username\n    id\n  }\n}\n\nfragment Hovercard_user on User {\n  id\n}\n\nfragment QuantcastCloudcastTracking_cloudcast on Cloudcast {\n  owner {\n    quantcastTrackingPixel\n    id\n  }\n}\n\nfragment UGCImage_picture on Picture {\n  urlRoot\n  primaryColor\n}\n\nfragment UserBadge_user on User {\n  hasProFeatures\n  isStaff\n  hasPremiumFeatures\n}\n\nfragment UserStreamPage_user on User {\n  id\n  displayName\n  username\n  stream(first: 10) {\n    edges {\n      cursor\n      repostedBy\n      node {\n        id\n        ...AudioCard_cloudcast\n        __typename\n      }\n    }\n    pageInfo {\n      endCursor\n      hasNextPage\n    }\n  }\n}\n\nfragment UserStreamPage_viewer on Viewer {\n  ...AudioCard_viewer\n}\n',
```

Comment:
You can put query string in template literals so that the indentation can be kept and is more readable